src/index.js

```diff
@@ -1,19 +1,57 @@
 // 1. Write a function that accepts an array of numbers. Return a new array with all elements doubled
 // Ex: double([1,2,3]) => [2,4,6]
+function doubleAll(element){
+  return element * 2;
+}
+arr = [ 1, 2, 3, 4, 5]
+let result = arr.map(doubleAll)
```

Comment:
The implementation is correct but just a few fix, the `doubleAll` function is supposed to receive the array, and these logic should be put inside the function